src/main/java/model/HighscoreEntry.java

```diff
@@ -0,0 +1,24 @@
+package model;
+
+public class HighscoreEntry {
+    
+    private final String name;
+    private final int score;
+    
+    HighscoreEntry(String name, int score) {
+        this.name = name;
+        this.score = score;
+    }
+    
+    public String getName() {
+        return name;
+    }
+    public int getScore() {
+        return score;
+    }
+    
+    @Override
+    public String toString() {
```

Comment:
Could you use the standard to string convention? Try deleting the method and press Alt + Insert to autogenerate.